packages/create-workleap/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as child_process from "child_process";
+import process from "process";
+
+import { Intro, Select, Note, Text, Outro } from "./prompts";
+import * as pkg from "./package.json";
+import { StartSpinner, StopSpinner } from "./spinner";
+
+const DEFAULT_OUTPUT_DIRECTORY = "./my-new-project";
+const NAME_PARAMETER_POSITION = 2; // TODO validate position of parameter once ask with `pnpm create`
+
+const FOUNDRY_CMD = "foundry";
+
+const AskForOutputDirectoryAsync = async (): Promise<string> => {
+  const outputDirectoryFromArgument: string =
+    process.argv[NAME_PARAMETER_POSITION];
+
+  if (outputDirectoryFromArgument) {
+    Note(`${outputDirectoryFromArgument} project setup`);
+  }
+
+  return (
+    outputDirectoryFromArgument ??
+    (await Text(
+      "Where should we create the project?",
+      DEFAULT_OUTPUT_DIRECTORY,
+      DEFAULT_OUTPUT_DIRECTORY
+    ))
+  );
+};
+
+const AskForTemplateAsync = (): Promise<string> => {
+  const templates = [
+    { value: "host-application" },
```

Comment:
could we list the directory in the templates folder instead? (should we?)

Reply:
When the `create-workleap` package will be published, the content of the `templates` folder will not be available (it's not part of the package, `degit` is how we will laod it instead). So we won't be able to list the templates.

A super warior way to determine the propmt quesqion can be from the `help` provided by the `@workleap/foundry` cli, but it will add more complexity. And with @patricklafrance we decided to simply duplicate the template information in this package, it's super simple to add a new template here.

Reply:
you're 100% right, my bad